ale_linters/eruby/erubylint.vim

```diff
@@ -0,0 +1,27 @@
+" Author: Matthias Guenther - https://wikimatze.de
+" Description: erb-lint for eruby/erb files
+
+function! ale_linters#eruby#erubylint#Handle(buffer, lines) abort
```

Comment:
I think the existing function in `ale_linters/ruby/ruby.vim` will handle these errors better. It has some slightly more complex code for handling column numbers. I recommend moving that existing handler to a new file `autoload/ale/handlers/ruby.vim`, renaming it appropriately for autoload semantics, and calling it from here. Update the tests appropriately.

That way, we'll be able to parse more errors here, and we'll get better test coverage.